kernel/common/process_mirror.rb

```diff
@@ -186,7 +186,6 @@ def convert_to_fd(obj, target)
             when 2
               if obj[0] == :child
                 fd = convert_to_fd obj[1], target
```

Comment:
With the check below being removed the assignment can also be removed (as far as I can tell).